bin/python/ripple/util/test_Sign.py

```diff
@@ -59,13 +52,17 @@ def test_make_manifest(self):
             '\xef*\x97\x16n<\xa6\xf2\xe4\xfb\xfc\xcd\x80P[\xf1s\x06verify')
 
     def test_sign_manifest(self):
+        ssk, spk = Sign.make_ecdsa_keypair(self.urandom)
```

Comment:
I can also attest to `test_Sign.py` tests failing when running `python -m unittest discover`